### PR TITLE
Add master-1.7 downgrade jobs

### PR DESCRIPTION
**Pull request by krzyzacy**
I'll make sure the configuration works, and I'll fill up 1.8-1.7 downgrade jobs

ref https://github.com/kubernetes/kubernetes/issues/52593

/assign @krousey @gmarek

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -2491,6 +2491,25 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-master-new-downgrade-cluster": {
+    "args": [
+      "--check-leaked-resources",
+      "--check-version-skew=false",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-new-master-upgrade-cluster.env",
```
Comment (krzyzacy) [inline jobs/config.json]: (intentionally reuse the same config as the upgrade env)

Comment (krousey) [reply jobs/config.json]: Does this set the skew variables appropriately to start at master and downgrade to 1.7?

Comment (krzyzacy) [reply jobs/config.json]: added `--skew` flag

**Comment by krzyzacy** [comment]
Also start with a parallel suite so we can gather results faster.

**Comment by krousey** [comment]
/lgtm

Though I still don't understand how the infrastructure knows which extracted target to start with. :smile:

**Comment by krzyzacy** [comment]
https://github.com/kubernetes/test-infra/blob/master/kubetest/extract.go#L133-L143 I think

File: jobs/config.json [hunk 2]
```diff
@@ -2491,6 +2491,26 @@
...
+      "--extract=ci/latest-1.7",
+      "--extract=ci/latest",
```
Comment (krousey) [inline jobs/config.json]: So these flag orders matter? and the -skew flag kind of reverses the way in which it matters?

Comment (krzyzacy) [reply jobs/config.json]: Yes the order of two --extract matters. I think --skew is to reverse which e2e binary to run.

**Pull request merged**